Django-Start-Project.md

```diff
@@ -0,0 +1,265 @@
+# Creating a Project with Django
+
+[Home](Python)
+
+Now that we know how to create virtual environments and use pip, we can begin
+building our project. In this article, we will start our first Django project,
+create tests, and start our development server.
+
+### Creating the Virtual Environment
+
+First, let's create a new virtual environment for this project. (If you haven't
+already, deactivate the previous virtualenv by typing `deactivate` in the
+terminal). Navigate to the directory where you want
+the Django project and type the following into the terminal:
```

Comment:
This would be a good place to link something from the [previous](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Install-Django-Flask) article; about how to activate or deactivate a virtual environment.